Tiltfile

```diff
@@ -1,34 +1,41 @@
 load("ext://k8s_attach", "k8s_attach")
 
-docker_build("controller", ".", only=("Dockerfile", "Makefile", "vendor","go.mod", "go.sum", "./api", "./cloud","./cmd", "./controller", "./util", "./version"), build_args={'VERSION': os.getenv("VERSION","")})
+docker_build(
+    "docker.io/linode/cluster-api-provider-linode",
+    context = ".",
+    only=("Dockerfile", "Makefile", "vendor","go.mod", "go.sum", "./api", "./cloud","./cmd", "./controller", "./util", "./version"),
+    ignore = ["templates"],
```

Comment:
my understanding of `only()` was it would ignore other dirs, are you seeing different behaviour requiring the ignoring of templates?